### PR TITLE
suppress ExternalDependencyExposedCheck for track 1

### PR DESCRIPTION
**Pull request by mssfang**
ExternalDependencyExposedCheck  will not check on track 1 libraries.

### CODE REVIEW

File: eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml [hunk 1]
```diff
@@ -85,4 +85,8 @@
 
   <!-- Add JavadocThrowsChecks suppression for any test or implementation files (ie. ones that live under src/test/java or implementation). -->
   <suppress checks="com.azure.tools.checkstyle.checks.JavadocThrowsChecks" files="(.*[/\\]src[/\\]test[/\\]java[/\\].*)|(.*[/\\]implementation[/\\].*)\.java"/>
```
Comment (conniey) [inline eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml]: The one above that also suppresses for "implementation" files. Seems reasonable to add to your suppression.

Comment (mssfang) [reply eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml]: This "implementation" already done in the code previously

Comment (conniey) [reply eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml]: This suppresses the check entirely, so it never has to load up your code.

Comment (mssfang) [reply eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml]: okay. add both ignore check for test and implementation package

**Pull request merged**